types/matter-js/matter-js-tests.ts

```diff
@@ -193,6 +193,8 @@ Composite.add(composite1, constraint1);
 Composite.add(composite1, mouseConstraint);
 // $ExpectType Composite
 Composite.add(composite3, [box1, composite2, constraint1, mouseConstraint]);
+// $ExpectType Composite
+Composite.remove(composite3, [box1, composite2, constraint1, mouseContraint]);
```

Comment:
Just a small typo, otherwise this looks good :)
```suggestion
Composite.remove(composite3, [box1, composite2, constraint1, mouseConstraint]);
```